services/director-v2/src/simcore_service_director_v2/modules/osparc_variables/substitutions.py

```diff
@@ -120,6 +121,7 @@ def create(cls, app: FastAPI):
             ("OSPARC_VARIABLE_NODE_ID", "node_id"),
             ("OSPARC_VARIABLE_PRODUCT_NAME", "product_name"),
             ("OSPARC_VARIABLE_STUDY_UUID", "project_id"),
+            ("OSPARC_VARIABLE_SERVICE_RUN_ID", "run_id"),
```

Comment:
ok.
TIP: did you know that now you can check the new registered osparc variable using the CLI?
```
$ cd services/director-v2
$ make install-dev
$ simcore-service osparc-variables 
OSPARC_VARIABLE_API_HOST
OSPARC_VARIABLE_API_KEY
OSPARC_VARIABLE_API_SECRET
OSPARC_VARIABLE_NODE_ID
OSPARC_VARIABLE_PRODUCT_NAME
OSPARC_VARIABLE_STUDY_UUID
OSPARC_VARIABLE_USER_EMAIL
OSPARC_VARIABLE_USER_ID
OSPARC_VARIABLE_USER_ROLE

```